lnrpc/lightning.proto

```diff
@@ -2564,9 +2564,17 @@ message PendingChannelsResponse {
 
         repeated PendingHTLC pending_htlcs = 8;
 
+        /*
+          There are three resolution states for the anchor:
+          limbo, lost and recovered. Derive the current state
+          from the limbo and recovered balances.
+        */
         enum AnchorState {
+            // The recovered_balance is zero and limbo_balance is non-zero
```

Comment:
Please always use complete sentences in comments (including full stops).

Reply:
Thank you for your review.
Slightly modified.